### DISCUSSION TITLE
Expiration of Github Copilot for Student

### DISCUSSION THREAD

**Discussion by Arijul420** [community/community#65951] [Copilot Conversations] [answered]
### Select Topic Area

General

### Body

Greeting! Hope everyone is doing well.
My github student developer pack will expire next month. Can I use github copilot after that for free? Will I need to buy Subscription to use copilot later?

**Comment by davecheney** [comment, marked as answer]
If you remain a student, you may re-apply for a new student developer pack. If you no longer qualify for a student developer pack then you will need to become a paid copilot user to continue to use the service.

**Comment by gauravjalap** [comment]
Can you tell me how you got to knew that when your student pack is expiring?  I can't see it anywhere.

**Reply by RafifR28** [reply]
Same too

**Reply by Wataru823** [reply]
I got an email that went something like this.

![CleanShot 2025-04-12 at 13 42 07@2x](https://github.com/user-attachments/assets/155e74b2-de7a-48ee-a061-7061c2fb847a)
